GripOpGras2.Server/Dockerfile

```diff
@@ -3,9 +3,6 @@
 FROM mcr.microsoft.com/dotnet/aspnet:6.0 AS base
 WORKDIR /app
 EXPOSE 4200
-EXPOSE 80
-ENV ASPNETCORE_URLS=http://+:4200
-
```

Comment:
Waarom heb  je deze wijzigingen (en de andere wijzigingen in dit betstand) gemaakt? Hierdoor werkt namelijk de port binding niet meer, en krijg ik de app niet runnend via docker

Reply:
okay het lijkt erop dat het niet meer uit maakt omdat je nu via die docker-compose werkt.